sda-commons-web-autoconfigure/src/test/java/org/sdase/commons/spring/boot/web/monitoring/testing/MetricsConstants.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2022- SDA SE Open Industry Solutions (https://www.sda.se)
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+package org.sdase.commons.spring.boot.web.monitoring.testing;
+
+import java.util.List;
+
+public final class MetricsConstants {
```

Comment:
Did you copy this class from dw-commons?
Or is it just used for the documentation of the actual metrics?
Could you add some documentation to the class to explain it?

Reply:
This is just a class to hold some constants fot the test, so the test class is not "polluted" with a lot of Strings. I added some documentation